### ISSUE TITLE
Quais são as bandeiras suportadas (ex: Visa, Master, Alelo)

### ISSUE THREAD

**Issue by magnocosta** [DeveloperCielo/LIO-SDK-API-Integracao-Remota-v1-Java#6]
Bom dia,

Nos exemplos que encontrei parece que não contempla todos os cartões, não consegui reconhecer todos os valores possíveis para as bandeiras.

Na classe [Card](https://github.com/DeveloperCielo/LIO-SDK-API-Integracao-Remota-v1-Java/blob/master/src/main/java/io/swagger/client/model/Card.java) temos um enum como abaixo:
```
public enum BrandEnum {
    VISA("VISA"),
    MASTERCARD("MASTERCARD"),
    ELO("ELO"),
    DINERS("DINERS"),
    AMEX("AMEX"),
    AURA("AURA");
}
```
Existem outras bandeiras que não estão presentes como Sorocred, Hipercard, Discover, Alelo entre outras. Fico em dúvida quais são os valores corretos que iram retornar para minha aplicação, seguem alguns exemplos:

EXPRESS retorna AMEX,
DINERS CLUB, retorna DINERS.

Seria possível adicionar a documentação todas as constantes possíveis.

**Obs:**
Criado por @frekele na issue #3, eu apenas extrai o conteúdo para essa issue na tentativa de deixar mais claro e fácil para as pessoas com a mesma dúvida.

Abs!

**Comment by frekele** [comment]
Boa Noite,

isso é importante, pois no lado cliente side tratamos isso como constantes e questionei qual o padrão definitivo das constates, como por exemplo hipercard,  eu não gostaria de ter que passar um cartão hipercard em produção para descobrir e depois atualizar em meu codigo, pois hipercard poderia retornar HIPER-CARD, HIPER_CARD, HIPERCARD ou HIPER por exemplo.

Também nesta classe não consta JCB, porem em outro componente existe: [order-management Card Class](https://github.com/DeveloperCielo/order-management/blob/master/order-management/src/main/java/cielo/lio/sdk/order/Card.java)

Seria interessante criarmos uma tabela na documentação com todas as bandeiras possíveis (independente de qual for o lojista credenciado), assim como temos no manual cielo-ecommerce:
https://developercielo.github.io/manual/cielo-ecommerce#produtos-e-bandeiras-suportadas

Estou a disposição para ajudar no que for necessário (testes, documentação e etc), caso queira.

Abraços

**Comment by magnocosta** [comment]
@frekele,

Concordo no ponto de que é ruim ter que observar um comportamento em produção para identificar o tipo de retorno. Eu não tenho essa informação mas solicitei aqui para que alguém me informe sobre isso.

**Comment by frekele** [comment]
@magnocosta ,

ok, então ficamos no aguardo.
